documentation/book/proc-deploying-cluster-operator-kubernetes-to-watch-multiple-namespaces.adoc

```diff
@@ -0,0 +1,75 @@
+// Module included in the following assemblies:
+//
+// assembly-cluster-operator.adoc
+
+[id='deploying-cluster-operator-kubernetes-to-watch-multiple-namespaces{context}']
+= Deploying the Cluster Operator to watch multiple namespaces
+
+.Prerequisites
+
+* Modify the installation files according to the namespace the Cluster Operator is going to be installed in.
++
+[source, subs="+quotes"]
+----
+sed -i 's/namespace: .\*/namespace: _<my-namespace>_/' examples/install/cluster-operator/*RoleBinding*.yaml
+----
+
+.Procedure
+
+. Modify the file `examples/install/cluster-operator/050-Deployment-strimzi-cluster-operator.yaml` and in the environment variable `STRIMZI_NAMESPACE` list all namespaces or projects where Cluster Operator should watch for resources.
+For example:
++
+[source,yaml]
+----
+apiVersion: extensions/v1beta1
+kind: Deployment
+spec:
+  template:
+    spec:
+      serviceAccountName: strimzi-cluster-operator
+      containers:
+      - name: strimzi-cluster-operator
+        image: strimzi/cluster-operator:latest
+        imagePullPolicy: IfNotPresent
+        env:
+        - name: STRIMZI_NAMESPACE
+          value: myproject,myproject2,myproject3
+----
+
+. For all namespaces or projects which should be watched by the Cluster Operator, install the `RoleBindings`.
+Replace the `_<my-namespace>_` or `_<my-project>_` with the namespaces or projects used in the previous step.
++
+ifdef::Kubernetes[]
+On {KubernetesName} this can be done using `kubectl apply`:
+[source,shell,subs=+quotes]
+kubectl apply -f examples/install/cluster-operator/020-RoleBinding-strimzi-cluster-operator.yaml -n _<my-namespace>_
+kubectl apply -f examples/install/cluster-operator/031-RoleBinding-strimzi-cluster-operator-entity-operator-delegation.yaml -n _<my-namespace>_
+kubectl apply -f examples/install/cluster-operator/032-RoleBinding-strimzi-cluster-operator-topic-operator-delegation.yaml -n _<my-namespace>_
++
+endif::Kubernetes[]
+On {OpenShiftName} this can be done using `oc apply`:
++
+[source,shell,subs=+quotes]
+oc apply -f examples/install/cluster-operator/020-RoleBinding-strimzi-cluster-operator.yaml -n _<my-project>_
+oc apply -f examples/install/cluster-operator/031-RoleBinding-strimzi-cluster-operator-entity-operator-delegation.yaml -n _<my-project>_
+oc apply -f examples/install/cluster-operator/032-RoleBinding-strimzi-cluster-operator-topic-operator-delegation.yaml -n _<my-project>_
+
+. Deploy the Cluster Operator
++
+ifdef::Kubernetes[]
+On {KubernetesName} this can be done using `kubectl apply`:
+[source,shell,subs=+quotes]
+kubectl apply -f examples/install/cluster-operator -n _<my-namespace>_
++
+endif::Kubernetes[]
+On {OpenShiftName} this can be done using `oc apply`:
++
+[source,shell,subs=+quotes]
+oc apply -f examples/install/cluster-operator -n _<my-project>_
+
+. Verify whether the Cluster Operator has been deployed successfully, the {KubernetesName} Dashboard or the command line:
++
+[source]
+----
+kubectl describe all
```

Comment:
I've said this before, but this step is meaningless unless you tell the user what to look for in the output.

Reply:
I have said it before as well. I have removed it.